wolfcrypt/benchmark/benchmark.c

```diff
@@ -3339,11 +3673,40 @@ void bench_rng(void)
                 remain -= len;
                 pos += len;
             }
+        #ifdef MULTI_VALUE_STATISTICS
+            if (runs == 0) {
+                delta = current_time(0) - start;
+                min = delta;
+                max = delta;
+            }
+            else {
+                delta = current_time(0) - prev;
+                if (max < delta)
+                    max = delta;
+                else if (min > delta)
+                    min = delta;
+            }
+
+            if (runs < MAX_SAMPLE_RUNS) {
+                deltas[runs] = delta;
+                sum += delta;
+                runs++;
+            }
+
+            prev = current_time(0);
+        #endif
```

Comment:
This lengthy code pattern repeats something like 125 times -- you should use a macro to orthogonalize the implementation.

The declarations of the working variables should also be done with a macro.

Reply:
Done.